### ISSUE TITLE
Configs via env vars 

### ISSUE THREAD

**Issue by MartinsThiago** [centrifugal/centrifugo#135]
The heroku demo app is configured over env vars. I tried to do the same locally but I didn't have success... Is it possible?  (I am using docker image) 

Also I saw there's no mention of env vars in the documentation

**Comment by MartinsThiago** [comment]
Here is my docker-compose config

```yaml
centrifugo:
  image: centrifugo/centrifugo:1.6.0
  environment:
  - CENTRIFUGO_SECRET=secret
  - CENTRIFUGO_WEB=1
  - CENTRIFUGO_ADMIN=1
  - CENTRIFUGO_ADMIN_PASSWORD=password
  - CENTRIFUGO_ADMIN_SECRET=secret
  ports:
    - "8569:8000"
```

**Comment by FZambia** [comment]
Hi, it's possible, this command works nice for me:
```
docker run --ulimit nofile=65536:65536 -e CENTRIFUGO_SECRET=mysecret -p 8000:8000 centrifugo/centrifugo centrifugo
```
Configuring via env is not flexible enough - as it's not possible to create channel namespace config. But maybe for your case it's OK.

I can't help you with compose at moment - what's the actual problem you came across? Env vars not set at all?

**Comment by MartinsThiago** [comment]
༼ᕗຈل͜ຈ༽ᕗ I think it worked

This is my first time using centrifugo, then I was first trying to access the admin page

I think that maybe viper is not getting boolean variables, but we can workaround it with `--flags`. Here are my changes

```yaml
centrifugo:
  image: centrifugo/centrifugo:1.6.0
  environment:
  - CENTRIFUGO_SECRET=potato
  - CENTRIFUGO_ADMIN_PASSWORD=potato
  - CENTRIFUGO_ADMIN_SECRET=potato
  command: centrifugo --web --admin
  ports:
    - "8569:8000"
```

Thks @FZambia 
...ᕕ( ᐛ )ᕗ

**Comment by FZambia** [comment]
You are welcome! Btw `--web` must automatically enable admin socket so you don't need to provide `--admin` option (as web useless without admin socket anyway) - but this is a bug in 1.6.0 - I'll fix it in 1.6.1
